service/src/main/kotlin/fi/espoo/evaka/attendance/MobileUnitController.kt

```diff
@@ -210,7 +226,11 @@ SELECT
     coalesce(pc.count, 0) AS present_children,
     coalesce(tc.count, 0) AS total_children,
     coalesce(ps.count, 0) AS present_staff,
-    coalesce(ts.count, 0) AS total_staff
+    coalesce(ts.count, 0) AS total_staff,
+    CASE
+        WHEN coalesce(ps.capacity, 0) = 0 THEN 'Infinity'::REAL
```

Comment:
Infinity is now shown even if there are no children present. Is that intentional?